### ISSUE TITLE
Sorry I still unkown the whole Train-Test workflow after reading all instructions.

### ISSUE THREAD

**Issue by Sewens** [github/CodeSearchNet#93]
Greetings, I'm a PhD student working in IR/SE. I'm very happy to see CSNet project and thank you for your great work.

I'm working on software representation and code retrieval/search. I'm familiar with IR dataset, but forgive me I quite not understand how CSNet dataset provide data annotataions.

I've got all six language for AWS, and I've found queries here [https://github.com/github/CodeSearchNet/blob/master/resources/queries.csv](url) in resource folder. 

I explore data from pandas.Dataframe and see all six language have been divided into train/test/valid partation label.

But I still cannot find the relation beweeen queries and code data. In other words, every query text has no related source code data for it. 

I'm newer to WandB platform, I wonder there be golden label for me to train model offline. Or I strictly need to write a dataloader or function for accepting WandB's online label data.

**Comment by mallamanis** [comment]
Hi @Sewens 
Does the response [here](https://github.com/github/CodeSearchNet/issues/91#issuecomment-561563233) reply to your comment maybe?

**Comment by Sewens** [comment]
I see. You means every query<-->code snippet relevance label can be seen as relationship (pair) between code documentation<-->main code body.

U means for model offline train/test we can only use code you provided. The ture label and negative sample need  We need to train a model that can select true code which match the documentation. The annotation will NOT be published, we can only evaluate model via WandB platform.

**Comment by mallamanis** [comment]
Exactly, the evaluation on the small human-annotated dataset can only be performed via the leaderboard. This is because the dataset is relatively small and it will be easy to overfit on it.

Having said that, you can use the `documentation<->code` as a proxy task as well as any other additional data you'd like (e.g. if you have some other data that you think can be useful during training/validation).
